view/home.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Peggy gou</title>
+    <link rel="stylesheet" href="/public/css/styles.css">
+</head>
+<body>
+    <nav>
+        <ul class="nav-list">
+            <li> <a href="/view/about.html" target="_blank"> About </a></li>
```

Comment:
Since now you have different `routes` for every `page` that you want in your website, you can make your `navbar` work, but for this to work the `path` that you need to specify in the `href` is the `url` path and not the path to the `html` file:

```
        <li><a href="/about"> About </a></li>
        <li><a href="/works">Works </a></li>
        <li><a href="/gallery">Photo Galery</a></li>
```